db/migrate/20220705144144_add_user_id_post_id_to_likes.rb

```diff
@@ -0,0 +1,6 @@
+class AddUserIdPostIdToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_column :likes, :user_id, :integer
```

Comment:
Please add `author_id` instead of `user_id` to likes table as it can be seen in this [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).